search-commons/src/test/java/no/unit/nva/indexingclient/utils/ShardRoutingUtils.java

```diff
@@ -0,0 +1,101 @@
+package no.unit.nva.indexingclient.utils;
+
+import static java.util.UUID.randomUUID;
+import static no.unit.nva.constants.Defaults.objectMapperWithEmpty;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.indexingclient.ShardRoutingService;
+import no.unit.nva.indexingclient.models.EventConsumptionAttributes;
+import no.unit.nva.indexingclient.models.IndexDocument;
+
+public class ShardRoutingUtils {
+  public static final String ID_FIELD = "id";
+  public static final String IDENTIFIER_FIELD = "identifier";
+  public static final String JOIN_FIELD = "joinField";
+  public static final String JOIN_FIELD_PARENT = "parent";
+  public static final String JOIN_FIELD_NAME = "name";
+  public static final String PART_OF = "partOf";
+  public static final String PLACEHOLDER_FIELD = "placeholder";
```

Comment:
Should these be public?

Reply:
Laziness? It should probably be refactored, but the reason was that I needed to use them in both test suites